docs/index.md

```diff
@@ -3,41 +3,266 @@ hide:
 ---
 
-# Home
+
+# Welcome to Taipy Documentation!
 
 Taipy is an innovative **low-code** package to create complete applications.
-It is composed of two main independent components:
-**Taipy GUI** and **Taipy Core**.
+This documentation is divided in three main sections:
+
+- [**"Getting Started"**](getting_started/index.md) provides a step-by-step opening to Taipy. Taipy features are leveraged as the application becomes more and more complex.
+- [**"User Manual"**](manuals/about.md) describes what the product concepts are, and how you can use them.
+- [**"Reference Manual"**](manuals/reference) gathers all the Python APIs that Taipy exposes.
+
+## Quick access
+
+These topics are the most visited ones, we thought you’d like to have a glance on them!
+
+<div class="tp-row tp-row--gutter-sm">
+  <div class="tp-col-12 tp-col-md-6 tp-col-lg-4 d-flex">
+    <a class="tp-content-card" href="getting_started/">
+      <img class="tp-content-card-icon icon-light" src="images/icons/flag-w.svg">
+      <img class="tp-content-card-icon icon-dark" src="images/icons/flag.svg">
+      <h3>Getting started</h3>
+      <p>
+        This tour shows you how to create an entire application using the two components of Taipy
+      </p>
+      <span class="tp-content-card-readmore">Read more</span>
+    </a>
+  </div>
+  <div class="tp-col-12 tp-col-md-6 tp-col-lg-4 d-flex">
+    <a class="tp-content-card" href="manuals/reference">
+      <img class="tp-content-card-icon icon-light" src="images/icons/menu_book-w.svg">
+      <img class="tp-content-card-icon icon-dark" src="images/icons/menu_book.svg">
+      <h3>Reference Manual</h3>
+      <p>
+        Description of Taipy’s API.
+      </p>
+      <span class="tp-content-card-readmore">Read more</span>
+    </a>
+  </div>
+  <div class="tp-col-12 tp-col-md-6 tp-col-lg-4 d-flex">
+    <a class="tp-content-card" href="manuals/gui/controls/">
+      <img class="tp-content-card-icon icon-light" src="images/icons/tune-w.svg">
+      <img class="tp-content-card-icon icon-dark" src="images/icons/tune.svg">
+      <h3>Controls</h3>
+      <p> Controls are UI objects for data vizualisation/interaction. </p>
+      <span class="tp-content-card-readmore">Read more</span>
+    </a>
+  </div>
+  <div class="tp-col-12 tp-col-md-6 tp-col-lg-4 d-flex">
+    <a class="tp-content-card" href="manuals/about/">
+      <img class="tp-content-card-icon icon-light" src="images/icons/dashboard-w.svg">
+      <img class="tp-content-card-icon icon-dark" src="images/icons/dashboard.svg">
+      <h3>Taipy’s user interface</h3>
+      <p>
+        The Taipy GUI library provides Python classes to create powerful Web apps in minutes.
+      </p>
+      <span class="tp-content-card-readmore">Read more</span>
+    </a>
+  </div>
+</div>
 
-Taipy requires *Python 3.8* or newer.
+## Browse dozens of controls to build your UI!
 
-<p align="center">
-  <img alt="Taipy structure" src="images/taipy-structure.svg">
-</p>
+<ul class="tp-pills-list">
```

Comment:
I love the hover effect here (tooltip)!
The big problem with the current implementation is that if we change the elements documentation, they cannot be updated in the doc. Which is bad.
Maybe these need to be generated after all.

Reply:
I'm not sure I understand the whole idea but I get the "must be generated" point. Let me know how you want to do that and if it needs my participation.

Reply:
I'm not sure I understand the whole idea but I get the "must be generated" point. Let me know how you want to do that and if it needs my participation.